src/__tests__/render.test.tsx

```diff
@@ -191,11 +195,34 @@ test('debug changing component', () => {
   // eslint-disable-next-line no-console
   const mockCalls = (console.log as any as ConsoleLogMock).mock.calls;
 
-  expect(stripAnsi(mockCalls[4][0])).toMatchSnapshot(
+  expect(stripAnsi(mockCalls[0][0])).toMatchSnapshot(
     'bananaFresh button message should now be "fresh"'
   );
 });
 
+test('debug with filterProps option', () => {
+  jest.spyOn(console, 'log').mockImplementation((x) => x);
+
+  const { debug, getByText } = render(<Banana />);
+
+  debug({ filterProps: (propName) => propName === 'children' });
```

Comment:
I'd either break those in different tests (to have better snapshot tests), and/or inline the snapshots. The snapshot are very "isolated" right now